content/en/archived-docs/corda-enterprise/4.6/enterprise/release-notes-enterprise.md

```diff
@@ -17,6 +17,14 @@ weight: 300
 
 # Corda Enterprise Edition 4.6 release notes
 
+## Corda Enterprise Edition 4.6.12 release notes
+
+Corda Enterprise Edition 4.6.12 is a patch release of Corda Enterprise focused on security improvements.
+
+### Upgrade recommendation
+
+As a developer or node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.9/enterprise/upgrading-index.md).
```

Comment:
Should this point to 4.10?

Reply:
Updated (Didn't use relref as it's in the archived-docs folder)